setup/artifacts/gitrepo.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: single
+title:  "Configure a Git Repo artifact account"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+This shows how to configure a Git Repo artifact account so Spinnaker can use entire repositories as a single artifact.
```

Comment:
```suggestion
This shows how to configure a Git Repo artifact account so Spinnaker can use an entire repository as a single artifact.
```

Reply:
Unless it's possible to (and plausible that one would want to) have an artifact comprising multiple repos.

Reply:
Nope, you're right, it would be just one 👍